merlin/main.py

```diff
@@ -68,33 +71,35 @@ def error(self, message):
         sys.exit(2)
 
 
-def verify_filepath(filepath):
+def verify_filepath(filepath) -> str:
     """
     Verify that the filepath argument is a valid
     file.
 
     :param `filepath`: the path of a file
```

Comment:
Do we need to add return typing to documentation?  I saw a couple other places they werent added so I wasnt sure.

Reply:
We should, I'm sure there are places I missed it. I'll make sure everything that was pushed with a ret val is declared.